src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -698,10 +701,10 @@ end
                                                                     1)
                                                                 for i in eachcomponent(equations))
 
-    rho = zero(cons_rho[1])
-    help1 = zero(cons_rho[1])
-    help2 = zero(cons_rho[1])
-    p = zero(cons_rho[1])
+    rho = 0
+    help1 = 0
+    help2 = 0
+    p = 0
```

Comment:
As written in another comment, this would have been a type instability in older versions of Julia. Can we maybe keep the previous version to clarify that it's fine for old people like me?


```suggestion
    rho = zero(eltype(cons_rho))
    help1 = zero(eltype(cons_rho))
    help2 = zero(eltype(cons_rho))
    p = zero(eltype(cons_rho))
```

Reply:
That's not old. But why not update to the most recent version of Julia? Some new packages actually rely on new Julia

Reply:
Sure - I'm typically using the latest stable release. However, we have also seen some issues, e.g., increased memory usage on newer versions of Julia impacting its use for high-performance computing. Thus, if there is no strong reason to drop support of older versions of Julia, I like to keep Trixi.jl (and other packages) compatible with them.

Reply:
Good.